### PR TITLE
Add functional test for manipulation plugin

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: manipulation/src/functTest/java/org/jboss/gm/manipulation/SimpleProjectFunctionalTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class SimpleProjectFunctionalTest {
+
+    @Rule
+    public TemporaryFolder tempDir = new TemporaryFolder();
+
+    @Test
+    public void ensureAlignmentFileCreated() throws IOException, URISyntaxException, XmlPullParserException {
```
Comment (metacosm) [inline manipulation/src/functTest/java/org/jboss/gm/manipulation/SimpleProjectFunctionalTest.java]: method name

File: manipulation/src/functTest/java/org/jboss/gm/manipulation/MultiModuleFunctionalTest.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class MultiModuleFunctionalTest {
+
+    @Rule
+    public TemporaryFolder tempDir = new TemporaryFolder();
+
+    @Test
+    public void ensureAlignmentFileCreated() throws IOException, URISyntaxException, XmlPullParserException {
```
Comment (metacosm) [inline manipulation/src/functTest/java/org/jboss/gm/manipulation/MultiModuleFunctionalTest.java]: Test name is incorrect.

File: manipulation/src/functTest/java/org/jboss/gm/manipulation/SimpleProjectFunctionalTest.java (hunk 1 above)
Comment (geoand) [reply manipulation/src/functTest/java/org/jboss/gm/manipulation/SimpleProjectFunctionalTest.java]: That's what I get for exercising  too much copy-pasta :laughing:

Comment (geoand) [reply manipulation/src/functTest/java/org/jboss/gm/manipulation/SimpleProjectFunctionalTest.java]: Fixed

File: manipulation/src/functTest/java/org/jboss/gm/manipulation/MultiModuleFunctionalTest.java (hunk 2 above)
Comment (geoand) [reply manipulation/src/functTest/java/org/jboss/gm/manipulation/MultiModuleFunctionalTest.java]: Fixed

**Pull request merged**